### ISSUE TITLE
CustomClasses doesn't support generic types

### ISSUE THREAD

**Issue by Tzrlk** [pwall567/json-kotlin-schema-codegen#15]
With the following property definition:
```json
...
"repositories": {
	"description": "The list of repositories being sourced.",
	"type": "object",
	"format": "map",
	"additionalProperties": {
		"$ref": "https://aetheric.co.nz/schema/gsm/repository"
	}
}
...
```
and the following config:
```json
"customClasses": {
	"format": {
		"map": "kotlin.collections.Map"
	}
}
```
I end up with:
```kotlin
import kotlin.collections.Map

data class Settings(
    val repositories: Map? = null
)
```
which is invalid.

If I add the generic signature to the type, it includes it in the import, which is also invalid.

I worked around the issue by creating a typealias and mapping to that, but I can see two ways to improve the experience:

* Strip off generic typing on the import declaration.
* Make `object` types without an explicit `name` or `$id` default to `Map<String,?>` where `?` is the common property type.

**Comment by pwall567** [comment]
Hi – very sorry not to have responded sooner.  Both your suggestions are good ones, but in the meantime, have you tried:
```
typealias MapStringAny = Map<String, Any>
```
Then, use `MapStringAny` in the `customClasses` configuration.

**Comment by Tzrlk** [comment]
Indeed, that's how I worked around it, so I really just wanted to document that.
